### DISCUSSION TITLE
Mods for gamepass dont work(Palworld)

### DISCUSSION THREAD

**Discussion by OkArcc** [UE4SS-RE/RE-UE4SS#400] [General]
So im pretty sure everything is in the right location but the mods dont work.
![pal modssssssss](https://github.com/UE4SS-RE/RE-UE4SS/assets/160520437/a0a4d60a-f11c-48d6-b19e-3cdd84cb20de)

**Comment by UE4SS** [comment]
Please create an [issue](https://github.com/UE4SS-RE/RE-UE4SS/issues/new/choose) instead and answer the questions in the automatically created description.
